context/OMEContext.tsx

```diff
@@ -25,9 +25,9 @@ export const parseOrders: (res: any) => Orders = (res) => {
         const sections = Object.values(orders);
         const flattenedOrders = sections.map((orders: any) =>
             orders.reduce(
-                (prev: any, order: { amount_left: number; price: number }) => ({
+                (prev: any, order: { remaining: number; price: number }) => ({
                     price: parseFloat(Web3.utils.fromWei(order.price.toString())), // price remains the same,
-                    quantity: prev.quantity + parseFloat(Web3.utils.fromWei(order.amount_left.toString())),
+                    quantity: prev.quantity + parseFloat(Web3.utils.fromWei(order.remaining.toString())),
```

Comment:
`amount_left` changing to `remaining` is unexpected, where did you see this?

Reply:
https://api.tracer.finance/book/1113ef6fcbbece1fa19155ba8deb223f88f5c266

Reply:
I created an order just then

Reply:
may have to ask Jack what it 'should' return

Reply:
oh I see, its `remaining` when fetching from the OME, but its stored in the database as `amount_left`. might be good to change one of them to be consistent

Reply:
definitely